src/pages/dashboard/analysis/components/TopSearch.tsx

```diff
@@ -0,0 +1,149 @@
+import { Icon as LegacyIcon } from '@ant-design/compatible';
+import { Card, Col, Row, Table, Tooltip } from 'antd';
+import { FormattedMessage } from 'umi-plugin-react/locale';
+import React from 'react';
+import numeral from 'numeral';
+import { SearchDataType, VisitDataType } from '../data.d';
+
+import { MiniArea } from './Charts';
+import NumberInfo from './NumberInfo';
+import Trend from './Trend';
+import styles from '../style.less';
+
+const columns = [
+  {
+    title: <FormattedMessage id="dashboardandanalysis.table.rank" defaultMessage="Rank" />,
+    dataIndex: 'index',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'dataIndex' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)